health-api/src/main/java/com/netflix/runtime/health/api/IndicatorFilters.java

```diff
@@ -0,0 +1,86 @@
+/**
+ * Copyright 2016 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.netflix.runtime.health.api;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Optional;
+
+/**
+ * Static builder of for creating an {@link IndicatorFilter} instance.
+ * 
+ * <pre>
+ *  IndicatorFilters.includes(someIndicatorInstance)
+ *                  .excludes("someIndicatorName")
+ *                  .build();
+ * </pre>
+ */
+public class IndicatorFilters {
+    
+    public static IndicatorFilterBuilder includes(String... indicatorNames) {
```

Comment:
Indicator names seems to restrictive here.  How about using a Matcher interface like this

```java
interface Matcher {
    boolean matches(HealthIndicator indicator);
}
```
 You can then have different matchers based on classes, packages, indicator names, existence of annotations, etc..